dump/mapping/bufr_adpupa_prepbufr.py

```diff
@@ -163,30 +163,30 @@ def _make_obs(comm, input_path, mapping_path, cycle_time):
 
     logging(comm, 'DEBUG', f'Do DateTime calculation')
     hrdr = container.get('variables/obsTimeMinusCycleTime')
-    hrdr_paths = container.get_paths('variables/obsTimeMinusCycleTime')
+    #hrdr_paths = container.get_paths('variables/obsTimeMinusCycleTime')
```

Comment:
Delete commented out code, this example and the following ones..

Reply:
@rmclaren thanks, deleted all the commented lines.